src/App.jsx

```diff
@@ -62,7 +64,7 @@ export function App() {
 
 	function logOut() {
 		const confirm = window.confirm(
-			`If you're ready to log out, make sure to write down your list name before you click ok! It is ${listToken}`,
+			`If you're ready to log out, make sure to write down your list name before you click ok! It is "${listToken}".`,
```

Comment:
I know we asked for something to make the list name distinguishable, but now I think the quotation marks may be copied together with the name and cause issues when pasting them in the input box and trying to join this list. On the other hand, the message asks the user to write the name down.
May be I am overcomplicating this.

Reply:
I'm going to leave it, if you're okay with that! I really like how it separates the text from the rest of the alert (and I think a user can figure it out?). Good looking out!